src/components/Select/Select.style.ts

```diff
@@ -14,11 +14,10 @@ export const selectWrapper =
       }
     `;
 
-export const rightIconContainer =
-  (isOpen: boolean, isSearchable: boolean) => (): SerializedStyles =>
-    css`
-      display: flex;
-      cursor: pointer;
-      transform: rotate(${isOpen && !isSearchable ? '180' : '0'}deg);
-      ${transition(0.2)}
-    `;
+export const suffixContainer = (isOpen: boolean, isSearchable: boolean) => (): SerializedStyles =>
```

Comment:
suffix/prefix containers can be one for all Field components? if this is one can we move it to shareable folder? what do you think?

Reply:
can we add them into the scope of the other ticket though before proceeding? 🙏